Screenshot Update for gist directory (#24041)
github/docs: content/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists.md

1. Sign in to {% data variables.product.product_name %}.
2. Navigate to your {% data variables.gists.gist_homepage %}.
3. Optionally, in the "Gist description" field, type a description for your gist.
4. In the "Filename including extension" field, type a file name for your gist, including the file extensions.
5. In the file contents field, type the text of your gist.
6. Optionally, to create {% ifversion ghae %}an internal{% else %}a public{% endif %} gist, click {% octicon "triangle-down" aria-label="The downwards triangle icon" %}, then click **Create {% ifversion ghae %}internal{% else %}public{% endif %} gist**.

>    ![Screenshot of the visibility dropdown menu for a new gist. Next to a button labeled "Create secret gist", a dropdown icon is outlined in dark orange.]{% ifversion ghae %}(/assets/images/help/gist/gist-visibility-drop-down-ae.png){% else %}(/assets/images/help/gist/gist-visibility-drop-down.png){% endif %}

Author: Hi @lecoursen, oh I missed that; thanks for pointing it out! The image has been updated now!

1. Sign in to {% data variables.product.product_name %}.
2. Navigate to your {% data variables.gists.gist_homepage %}.

> 3. In the "Gist description" field, type an optional description for your gist.

Reviewer: https://github.com/github/docs-internal/blob/main/contributing/content-style-guide.md#procedural-steps

```suggestion
3. Optionally, in the "Gist description" field, type a description for your gist.
```

4. In the "Filename including extension" field, type a file name for your gist, including the file extensions.
5. In the file contents field, type the text of your gist.
6. Optionally, to create {% ifversion ghae %}an internal{% else %}a public{% endif %} gist, click {% octicon "triangle-down" aria-label="The downwards triangle icon" %}, then click **Create {% ifversion ghae %}internal{% else %}public{% endif %} gist**.

> ![Screenshot of a visibility drop-down menu on GitHub Gist. A little dropdown icon beside a button labelled "Create secret gist" is outlined in dark orange.]{% ifversion ghae %}(/assets/images/help/gist/gist-visibility-drop-down-ae.png){% else %}(/assets/images/help/gist/gist-visibility-drop-down.png){% endif %}

Reviewer: The border on this one still looks too thin to me. 

The alt text is great though! Just a few tiny suggestions to make it slightly easier to read, and use American spelling. 

```suggestion

   ![Screenshot of the visibility dropdown menu for a new gist. Next to a button labeled "Create secret gist", a dropdown icon is outlined in dark orange.]{% ifversion ghae %}(/assets/images/help/gist/gist-visibility-drop-down-ae.png){% else %}(/assets/images/help/gist/gist-visibility-drop-down.png){% endif %}
```

> ![Drop-down menu to select gist visibility]{% ifversion ghae %}(/assets/images/help/gist/gist-visibility-drop-down-ae.png){% else %}(/assets/images/help/gist/gist-visibility-drop-down.png){% endif %}

Reviewer: I don't think the `gist_name_description` and `gist_test_box` images are necessary. When you navigate to the Gist homepage, there was only 3 fields. Folks should be able to find them by description alone. 

Instead, let's remove both those images and update the procedural steps to clarify which fields we're talking about. I can't leave this comment on the appropriate lines unfortunately 😅 but something like this:

1. In the "Gist description" field, type a description for your gist.
2. In the "Filename including extension" field,  type a file name for your gist, including the file extensions.
3. In the file contents field, type the text of your gist.

> ![Drop-down menu to select gist visibility]{% ifversion ghae %}(/assets/images/help/gist/gist-visibility-drop-down-ae.png){% else %}(/assets/images/help/gist/gist-visibility-drop-down.png){% endif %}

Reviewer 2: 👍
